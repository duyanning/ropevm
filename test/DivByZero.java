
class DivByZero{
	public static int zero=0;
	public static void main(String[] args){
		try{
			int x = 1/zero;
			System.out.println("NO DIVISION BY ZERO EXCEPTION");
		}catch(ArithmeticException e){
			System.out.println("Caught division by zero exception");
		}
	}
}

