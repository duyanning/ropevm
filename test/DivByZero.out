Caught division by zero exception
